fonts/Makefile.am

```diff
@@ -120,6 +120,10 @@ dist_font_DATA = \
 	Skranji-Regular.ttf \
 	Sniglet-ExtraBold.ttf \
 	Sniglet-Regular.ttf \
+	SutonnyMJ.ttf \
+	SutonnyMJ-Bold.ttf \
+	SutonnyMJ-BoldItalic.ttf \
+	SutonnyMJ-Italic.ttf \
```

Comment:
That's all I have. I've requested the license file to our partners from mPower, because I couldn't find it online. I'll add it once I receive it.